dev/tests/behavioral/features/front/product-purchase.feature

```diff
@@ -0,0 +1,62 @@
+@javascript
```

Comment:
Sorry, but unfortunately this commit 9e3faf4d18e9d078c7f5416627e98ee90d9ad641 failed to pass all tests. 1 features failed.
Failed suites:
- Behat tests [1]
http://localhost/test-run/317/